unittests/wasm_tests.cpp

```diff
@@ -1843,6 +1845,39 @@ BOOST_FIXTURE_TEST_CASE( depth_tests, TESTER ) try {
 
 } FC_LOG_AND_RETHROW()
 
+// TODO: Update to use eos-vm once merged
+BOOST_FIXTURE_TEST_CASE( code_size, TESTER )  try {
+   using namespace IR;
+   using namespace Runtime;
+   using namespace Serialization;
+   std::vector<U8> code_start = {
+      0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00, 0x01, 0x07, 0x01, 0x60,
+      0x03, 0x7e, 0x7e, 0x7e, 0x00, 0x03, 0x02, 0x01, 0x00, 0x07, 0x09, 0x01,
+      0x05, 0x61, 0x70, 0x70, 0x6c, 0x79, 0x00, 0x00, 0x0a, 0x8b, 0x80, 0x80,
+      0x0a, 0x01, 0x86, 0x80, 0x80, 0x0a, 0x00
+   };
+
+   std::vector<U8> function_body = {
+      0x41, 0xff, 0xff, 0xff, 0xff, 0x07, 0x1a
+   };
```

Comment:
It would be simpler (and easier to verify that the counts are correct) if you just filled the function body with nop (0x01).